src/controllers/EvidenceDownloadController.ts

```diff
@@ -0,0 +1,45 @@
+import { AxiosResponse } from 'axios';
+import { Response } from 'express';
+import { OK } from 'http-status-codes';
+import { inject } from 'inversify';
+import { controller, httpGet, requestParam } from 'inversify-express-utils';
+import { BaseAsyncHttpController } from './BaseAsyncHttpController';
+
+import { FileTransferService } from 'app/service/FileTransferService';
+import { DOWNLOAD_FILE_PAGE_URI } from 'app/utils/Paths';
+
+const template = 'download-file';
+
+@controller(DOWNLOAD_FILE_PAGE_URI)
+export class EvidenceDownloadController extends BaseAsyncHttpController {
+
+    constructor(@inject(FileTransferService) private readonly fileTransferService: FileTransferService) {
+        super();
+    }
+
+    @httpGet('/prompt/:fileId')
```

Comment:
Do we have that prompt screen in prototype? If so what is the URL?

Reply:
`host:port/download`

Reply:
Thank you